src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
```

Comment:
Here you can use destructuring to improve code readability.

Usage example:

```suggestion
  <section className="Person">
    const { name, age, sex, isMarried, partnerName } = person;
    
    <h2 className="Person__name">{`My name is ${name}`}</h2>
```